app/actions/Workspace/dashboardHeader.js

```diff
@@ -19,15 +19,35 @@ const localnodeData = {
 
 function checkIsLocalNodeRunning() {
   return new Promise((resolve) => {
-    exec(
-      'tezster get-balance tz1KqTpEZ7Yob7QbPE4Hy4Wo8fHG8LhKxZSx',
-      (err, stdout) => {
-        if (err || stdout.split('ECONNREFUSED').length > 1) {
-          return resolve(false);
+    if (process.platform.split('win').length > 1) {
+      const ls = spawn(
+        'cmd.exe',
+        [
+          '/c',
+          'powershell.exe tezster get-balance tz1KqTpEZ7Yob7QbPE4Hy4Wo8fHG8LhKxZSx',
```

Comment:
move this wallet address to a constant file and use that contstant everywhere so it's easy to update.